src/cart/SConscript

```diff
@@ -54,8 +53,10 @@ def scons():
 
     Export('env', 'prereqs', 'CART_VERSION')
     # generate targets in specific build dir to avoid polluting the source code
+    SConscript('../gurt/SConscript')
```

Comment:
These should be referenced from src/SConscript instead

Reply:
I tried to do this at the beginning but noticed I had to move the 'CART_VERSION' variable out of the "src/cart/SConscript" to the upper "src/SConscript" file so I decided to leave the "../gurt/SConscript" call in this file.

I just made the updates and moved the above and seems to compile and work fine. Let me know if you consider this is Ok, otherwise I will roll-back these changes.

Thank you.